### PR TITLE
Add interfaces/largest-contentful-paint.idl

### PR DESCRIPTION
**Pull request by autofoolip**
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/d3e2b3f/whatwg/idl/largest-contentful-paint.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/563471075

### CODE REVIEW

**Comment by lukebjerring** [comment]
@foolip - added an idlharness test, PTAL

**Comment by foolip** [comment]
@lukebjerring oh noes, race condition: https://github.com/web-platform-tests/wpt/pull/18082

File: largest-contentful-paint/idlharness.window.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+// https://wicg.github.io/largest-contentful-paint/
+
+'use strict';
+
+idl_test(
+  ['largest-contentful-paint'],
+  ['performance-timeline', 'dom'],
+  idl_array => {
+    idl_array.add_objects({
+      // LargestContentfulPaint: [ TODO ]
```
Comment (foolip) [inline largest-contentful-paint/idlharness.window.js]: This is done in https://github.com/web-platform-tests/wpt/pull/18082 :)

**Comment by lukebjerring** [comment]
Done better, even.

**Pull request closed without merging**